### DISCUSSION TITLE
Project Homepage

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#8435] [Feature Requests]
I would like admins to have the ability to promote any page within Directus to be a "homepage". This page would be used instead of a 404 page, and in case the user doesn't have a _ref_ or _last_page_ set.

I imagine the default would be the initial Insights Dashboard (which we still need to create), showing global usage, recent items, etc.

**Comment by licitdev** [comment]
Yes!! Hopefully the homepage setting will be based on role based and that it can be set to any relative url.
In addition, a toggle to force home page could be useful, adding the ability to ignore `last_page`.
So the user will no longer be redirected to a collection item detail page right after login.

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
